core/src/com/po/fuck/model/Room.java

```diff
@@ -134,4 +137,8 @@ public void update(float delta) {
             throw new AssertionError("Unknown state");
         }
     }
+
+    public float getElapsedTime() {
+        return elapsedTime;
```

Comment:
Why is it here ?

Reply:
For example, a bit of animation of the room, possibly several frames with minimal changes. Hm. Probably would be better to add `elapsedTime` and `getElapsedTime` to `Drawable`, because we almost always need this information during drawing process.

Reply:
> because we almost always need this information during drawing process.

So move it to the view.